app/models/user.rb

```diff
@@ -86,6 +86,39 @@ class User < ActiveRecord::Base
     },
     ranked_by: ":tsearch + (0.25 * :trigram)"
 
+  def self.dormant(window=5)
```

Comment:
5 days is really not that long, a short break away. This window really should be expanded to at least 2 weeks and probably 1 month. Thoughts?